lte/gateway/python/magma/pipelined/ebpf/ebpf_manager.py

```diff
@@ -201,7 +215,7 @@ def add_ul_entry(self, mark: int, ue_ip: str):
         )
 
         key = self.ul_map.Key(ip_addr)
-        val = self.ul_map.Leaf(mark, self.sgi_if_index, self.ul_src_mac, self.ul_gw_mac, 0)
+        val = self.ul_map.Leaf(mark, self.sgi_if_index, self.ul_src_mac, self.ul_gw_mac, b'0')
```

Comment:
I have no idea why it wants bytes here for the `uint64_t bytes` but the same variable for dl_map works fine with just `0`
@pshelar any ideas

Reply:
what us error you are seeing?

Reply:
Expected bytes got int

Reply:
Looks like bcc/python issue, following is work around:
```
diff --git a/orc8r/gateway/c/common/ebpf/EbpfMap.h b/orc8r/gateway/c/common/ebpf/EbpfMap.h
index 250af6c1b..95a6baa36 100644
--- a/orc8r/gateway/c/common/ebpf/EbpfMap.h
+++ b/orc8r/gateway/c/common/ebpf/EbpfMap.h
@@ -24,9 +24,9 @@ struct ul_map_key {
 struct ul_map_info {
   __u32 mark;
   __u32 e_if_index;
+  uint64_t bytes;
   __u8 mac_src[ETH_ALEN];
   __u8 mac_dst[ETH_ALEN];
-  uint64_t bytes;
 };

 struct dl_map_key {
```